setup.py

```diff
@@ -24,7 +24,7 @@
       extras_require={
           'pipfile': ['pipreqs', 'requirementslib'],
           'pyproject': ['toml'],
-          'requirements': ['pip', 'pipreqs'],
+          'requirements': ['pip', 'pipreqs', 'pip-api'],
```

Comment:
pip dependency can be removed?  It's not imported anywhere.

Reply:
Good call! I've removed it: https://github.com/timothycrosley/isort/commit/d8899b9e0ac6d928c1e91b7ea1eefb2913f3c704